File: python/easy/difference_of_sums.py
def t(n,m):
    num1 = 0
    num2 = 0
    for num in range(1,n+1):
        if num % m == 0:
            num2 += num
        else:
            num1 += num
    return num1 - num2
print(t(10,3))

File: ts/easy/differenceOfSums.ts
function differenceOfSums(n: number, m: number): number {
  const num1 = new Array(n)
                  .fill('')
                  .map((item, i) => item = i+1 )
                  .filter((num) => num % m !== 0)
                  .reduce((a, b) => a + b, 0);
  const num2 = new Array(n)
                  .fill('')
                  .map((item, i) => item = i+1 )
                  .filter((num) => num % m === 0)
                  .reduce((a, b) => a + b, 0);
  return num1 - num2;
};